mentoring/tip.py

```diff
@@ -64,7 +64,7 @@ def render(self):
         """
         Returns a fragment containing the formatted tip
         """
-        fragment, named_children = self.get_children_fragment({})
+        fragment, named_children = self.get_children_fragment({ContextConstants.AS_TEMPLATE: False})
```

Comment:
@e-kolpakov @antoviaque I had to do this because otherwise HTML content in tooltips would get rendered as a template and the JS would insert the HTML into a wrong position in the DOM (to the `.feedback` element rather than inside `.tip-choic-group`). I assume this is safe for tooltips, but I don't really understand why exactly JS templates are needed in some cases, so please let me know if this is problematic.